### PR TITLE
Removed Elasticsearch, Updated Archtiecture.png

### LINKED ISSUE #125: Add draw.io architecture diagram
## Type of Issue:

- [ ] Bug: Something isn't working
- [x] Enhancement: New feature or request
- [ ] Question: Further information is requested (I have a question)

#### Issue Creator Checklist

- [x] The appropriate label (bug, enhancement, question) has been attached to this issue.
- [x] This issue has been thoroughly documented below; a developer should be able to understand the issue by reading it.

## Background

There is a project wide push to move to draw.io for architecture diagrams.  Moving to an opensource diagramming solution should help enable anyone to update diagrams.
This issue is to add a draw.io architecture diagram of the quickstart to the repo.  The existing diagram is a static image export from LucidChart.  Copying and pasting the source file to draw.io should accomplish much of the work here.
Draw.io files are exported in XML.  It would be ideal if the draw.io export, when added to git, could be viewed from the README.  This would replace the need to have a static image checked in.
If the draw.io XML cannot be rendered from the README, the static image should still be removed.  It will undoubtedly fall out of date.  If this is the case, perhaps a github actions should publish the diagram on every push?  And a link in the README can take a user to where the diagram was published?
There's room to pick and choose here.  There's also static image editing, explained here: https://desk.draw.io/support/solutions/articles/16000042371-embed-a-diagram-in-github-markdown
The Pull Request template should be updated, too, with a checklist item for "I have updated the diagram if appropriate".

## Assorted Notes/Considerations

This is an implementation of work first done on BigMac.  See https://github.com/CMSgov/cms-bigmac/pull/38 for details.

## AC:

- An svg draw.io architecture diagram of the quickstart is added to the repository
- The existing .png diagram is removed
- A solution to directly render (in README) or link to the diagram is implemented.
- The pull request template is updated; a checklist item is added for updating the diagram, if appropriate.

### PR DESCRIPTION
**Pull request by ayoneeyee**
## Purpose

This issue is to remove all components ElasticSearch from the quickstart. While there's nothing wrong with the ES implementation here, and while it demonstrates how to have another system plug into a data stream, it's slowing MAPro down. The projects that fork this repository are not in the market for ES. And if they were, the ES service would (hopefully) be a separate project that plugs into any number of data streams from Kafka (instead of DynamoDB Streams). The SES emailer functions are seen as a sufficient example of how to build event driven business logic while being lightweight; these emailers should remain. The ES service is a bigger and more expensive example of this, and isn't worth it for us.

#### Linked Issues to Close

Closes https://github.com/CMSgov/macpro-quickstart-serverless/issues/131
Closes #125 

## Approach

All ElasticSearch services, components, and support has been removed.
Some of the files removed include:
services/elasticsearch-auth/package.json
services/elasticsearch-auth/yarn.lock
services/elasticsearch/package.json
services/elasticsearch/yarn.lock
services/stream-functions/package.json
services/stream-functions/yarn.lock

## Learning

Removed elasticsearch from the quickstart serverless repo. Then also removed all references of elasticsearch from other files and directories in the following repo. 
Reviewed logs to make sure there are no error and build completed successfully.
https://github.com/CMSgov/macpro-quickstart-serverless/tree/master/services

## Assorted Notes/Considerations
Make sure to review Github Action logs in details for any issue.

Existing environment will need Elasticsearch and Elasticsearch-auth deleted manually.

Updated the JIRA Ticket of updates and changes

#### Pull Request Creator Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [x] This PR meets all acceptance criteria for those issues.
- [x] This PR and linked issue(s) are adequately documented
- [x] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.
- [x] Someone has been assigned this PR.
- [x] At least one person has been marked as reviewer on this PR.

#### Pull Request Reviewer/Assignee Checklist

- [x] This PR has an associated issue or issues.
- [x] The associated issue(s) are linked above.
- [ ] This PR meets all acceptance criteria for those issues.
- [ ] This PR and linked issue(s) are adequately documented
- [ ] This PR and linked issues(s) are a complete description of the changeset; an individual or team should be able to understand the issue(s) and changes by reading through this PR and it's links, with no further interaction.

### CODE REVIEW

**Comment by mdial89f** [comment]
@ayoneeyee update your branch with master

**Comment by mdial89f** [comment]
@ayoneeyee please see earlier comment. 
In addition, please add this PR to the project. Theres a check list item that intends to ensure PRs get added to the project board, which you checked, but the PR isn't added to the project

**Review by mdial89f — CHANGES_REQUESTED** [review]
@ayoneeyee 
1. You must merge master into your branch before this can be fully reviewed; see earlier comment.
2. Please put somewhere in your PR, probably in the Notes section, that existing elasticsearch and elasticsearch-auth services should be deleted by a developer post-merge.  They won't clean up on their own, and it's important that consumers of this PR understand this point.
3. Please add closing language to the linked issues section; this PR closes the "update draw.io diagram issue".
4. Please put that draw.io issue in progress.

File: services/stream-functions/package.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   "name": "stream-functions",
   "description": "",
   "version": "1.0.0",
-  "dependencies": {
-    "dynamo-to-elasticsearch-nodejs": "^1.0.5",
-    "dynamodb-stream-elasticsearch": "^2.11.1"
-  },
-  "devDependencies": {}
```
Comment (mdial89f) [inline services/stream-functions/package.json:9]: The devDependencies portion of this merge is incorrect.
The devDependencies value should not be changed for this PR; it should be as it was, {}.

File: services/stream-functions/serverless.yml [hunk 2]
```diff
@@ -31,49 +31,9 @@ custom:
...
-          arn: ${self:custom.tableStreamArn}
-          startingPosition: LATEST
-          maximumRetryAttempts: 2
-    role: LambdaElasticSearchIndexerRole
-    environment:
-      ES_ENDPOINT_URL: https://${self:custom.elasticSearchDomainEndpoint}
-      INDEX: amendments
-      TYPE: json
-    maximumRetryAttempts: 2
-  emailSubmitter:
```
Comment (mdial89f) [inline services/stream-functions/serverless.yml:53]: This function should not be removed.  It isn't related to elasticsearch.

File: services/stream-functions/serverless.yml [hunk 3]
```diff
@@ -31,49 +31,9 @@ custom:
...
-    events:
-      - stream:
-          arn: ${self:custom.tableStreamArn}
-          startingPosition: LATEST
-          maximumRetryAttempts: 2
-    role: LambdaEmailerRole
-    environment:
-      emailSource: ${self:custom.sesSourceEmailAddress}
-    maximumRetryAttempts: 2
-  emailReviewer:
```
Comment (mdial89f) [inline services/stream-functions/serverless.yml:64]: This function should not be removed.  It isn't related to elasticsearch.

**Review by mdial89f — CHANGES_REQUESTED** [review]
I left 3 comments on files.
In addition to addressing those, note that the style_checks (prettier) workflow is failing for your PR.  You must run prettier and commit/push its style changes.
To run prettier
`npx prettier --write .`

File: services/stream-functions/handlers/elasticsearchIndexer.js [hunk 4]
```diff
@@ -1,5 +1,3 @@
-const { pushStream } = require("dynamodb-stream-elasticsearch");
```
Comment (mdial89f) [inline services/stream-functions/handlers/elasticsearchIndexer.js:1]: @ayoneeyee This file should be removed entirely

**Review by mdial89f — CHANGES_REQUESTED** [review]
@ayoneeyee what happened to the architecture diagram update?

File: services/stream-functions/serverless.yml (hunk 2 above)
Comment (dwhitecl) [reply services/stream-functions/serverless.yml:53]: lines 40-52 is the only function that should be removed

**Pull request closed without merging**